### DISCUSSION TITLE
Localized Symfony Routes with same structure as Pimcore sites

### DISCUSSION THREAD

**Discussion by TS1997** [pimcore/pimcore#10472] [Q&A / Support] [answered]
Hello,

I'm currently building a site in Pimcore 10 that I've setup to use Pimcore sites for the different languages. I've set these up so the home URL for each language is http:/localhost/(en|se|fr) in the dev environment and http://example.(com|se|fr) in production. I'm now wondering how I should go about defining the Symfony routes to work with this setup. 

I currently have them set up like this:
`@Route("/{_locale}/account/index", name="account-index")`
This works fine in the dev-environment that always has the locale as a base "folder" but how would I set these up to also work in the production environment where the domain is responsible for the locale rather than a "folder"? The current setup with the example above only works with http://localhost/sv_SE/account/index or http://example.se/sv_SE/account/index but I want it to work with http://localhost/sv_SE/account/index and http://example.se/account/index

Is this possible since it's possible with sites and how would I go about doing this if so?

Thanks in advance!

**Comment by dpfaffenbauer** [comment]
You could use Hardlinks for that. But: If you have the same website available as example.com/sv_SE and example.se you run into SEO Problems....

**Reply by TS1997** [reply]
I don't want to keep example.com/sv_SE. I want a solution where the domain dictates the language. So example.com will be the English site and if you change to Swedish you'll automatically go to example.se instead of example.com/sv_SE.

**Comment by dpfaffenbauer** [comment]
then why not create sites in pimcore for each language?

**Reply by TS1997** [reply]
I've already done that and that part works fine. I'm currently trying to find a solution where I can get regular Symfony route annotations to work in the same manner as every page won't be a Pimcore document. Some will be a Symfony route that renders a specific template and I can't seem to find a solution that allows me to have the same URL structure for both

**Comment by TS1997** [comment, marked as answer]
I've now managed to solve this issue by creating custom route configs for the different environments
